docs/install.md

```diff
@@ -108,7 +108,7 @@ Require it with an explicit version constraint allowing its desired stability.
 There are a few setup items to do before you can start using Shield in
 your project.
 
-1. Copy the **Auth.php** and  **AuthGroups.php** from **vendor/codeigniter4/shield/src/Config/** into your project's config folder and update the namespace to `Config`. You will also need to have these classes extend the original classes. See the example below. These files contain all the settings, group, and permission information for your application and will need to be modified to meet the needs of your site.
+1. Copy the **Auth.php**, **AuthGroups.php**, and **AuthToken.php** from **vendor/codeigniter4/shield/src/Config/** into your project's config folder and update the namespace to `Config`. You will also need to have these classes extend the original classes. See the example below. These files contain all the settings, group, and permission information for your application and will need to be modified to meet the needs of your site.
```

Comment:
This change is good. 
But [Command Setup](https://codeigniter4.github.io/shield/install/#command-setup) does not copy `AuthToken.php`. 
> Run the following command. This command handles steps 1-5 of Manual Setup and runs the migrations.

Reply:
I have added the copy to the setup script and updated the tests scripts accordingly.

Reply:
Thank you. It worked.